### PR TITLE
ci: adding Matthieu as approver for ADEV

### PR DESCRIPTION
**Pull request by JeanMeche**
This is to help the team approve changes on the new doc site.

### CODE REVIEW

**Review by twerske — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
nit (really minor!): could you please update the commit message to say "ci: adding Matthieu as approver for ADEV", so that it's easier to navigate commit list later?

**Comment by JeanMeche** [comment]
Sure & ✅

File: .pullapprove.yml [hunk 1]
```diff
@@ -1083,6 +1083,7 @@ groups:
         - crisbeto
         - atscott
         - pkozlowski-opensource
+        - ~jeanmeche
```
[the remark this answers is not in the document]
Comment (JeanMeche) [reply .pullapprove.yml:1086]: The `~` denotes a silent approver. I can approve but won't be part of the default pool !

[the remark this answers is not in the document]
Comment (AndrewKushnir) [reply .pullapprove.yml:1086]: Oh, I see, great 👍

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
**Caretaker note**: current set of approvals should be sufficient, this PR is ready for merge.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 35e5ba0ba6bfb2bd0ce96caf2f311bfd7d954e49.

**Pull request closed without merging**